live-event-app/README.md

```diff
@@ -0,0 +1,36 @@
+# Live Event App
+
+The purpose of this application is to simulate possible use cases that require iOS Chat Components to wokr under heavy load.
+
+## Prerequisites
+
+| Name | Requirement |
+| :--- | :------ |
+| [Xcode](https://developer.apple.com/xcode/resources/) | >= 13.0 |
+| [npm](https://www.npmjs.com/) | >= 19.1.0 |
+
+## Usage
+
+1. Install the `npm` package manager. For instance, you can do it using [Homebrew](https://brew.sh/):
+
+```
+brew install npm
+```
+
+1. In the Terminal application, go to the `tomato` directory and execute `npm install`
+
+2. Run the mock server by executing `npx tomato` in the `tomato` directory
+
+3. See the [contracts list](https://github.com/pubnub/chat-specifications/tree/master/Contracts) with their names and purposes
```

Comment:
this branch is on public repo and chat-specifications is on private repository, I am not sure that we should publish informations about our internal work like tomato and private repos.

Reply:
Alright, then I assume we know how to load `tomato` from the `live-event-app` level and how to pass the given contract. If yes, I will delete this description.

Reply:
It is not my call. Is there an option where user connects to live server and not to tomato?